platform/linux-generic/odp_crypto.c

```diff
@@ -20,6 +20,9 @@
 #include <odp/api/plat/packet_inlines.h>
 #include <odp_packet_internal.h>
 
+/* Inlined API functions */
+#include <odp/api/plat/event_inlines.h>
```

Comment:
I had a different approach to this issue. It is quite easy to forget to actually include the `_inlines.h` header file. My suggestion is to create a set of `<odp_FOO.h>` headers used only in platform code, which include both `<odp/api/FOO.h>` and `<odp/api/plat/FOO_inlines.h>`. Platform code will always include the former header, which will pull inlineable functions.

Reply:
@lumag That would also have the advantage that new functions can be inlined without requiring any changes in those modules that make use of the newly-inlined functions. So these can be optimized individually without chasing who uses them.

Reply:
Bill, in both approaches each c file would need only one change: include odp_foo.h or odp/api/plat/foo_inlines.h.

I noticed odp_foo.h, but I think it's name should tell that it's used for getting access to inline implementation (so odp_foo_inlines.h). When foo module includes already odp/api/foo.h, it unnecessary to create another file that include the same again (plus inlines.h). Also I think it's more readable (and flexible) to include in each c file explicitly include those headers and only those headers it needs. Also we might not want to inline every API in every c file. This explicit include of odp/api/plat/foo_inlines.h documents when a c file uses foo inlines, and when it does not use those.

Reply:
@psavol fine, let's see, how this will work.